src/keystore.c

```diff
@@ -427,6 +427,20 @@ static bool _get_xprv(const uint32_t* keypath, const size_t keypath_len, struct
     return true;
 }
 
+bool get_root_fingerprint(char* fingerprint, int fingerprint_len)
+{
+    static const uint32_t keypath[] = {
+        0 + BIP32_INITIAL_HARDENED_CHILD,
```

Comment:
this is `m/0'`, but that is not the root? `m/` is. any reasoning I am missing?

Reply:
No, just me getting the naming wrong. How about `get_accounts_fingerprint`?

Reply:
On the other hand it does get the fingerprint of the parent public key of `m/0'`. That should then be the first four bytes of `m/`, right?

Reply:
https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki#key-identifiers

first 4 bytes of hash160 is the key fingerprint. children also encode the parent key fingerprint (see [here](https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki#serialization-format)).

To get the root fingerprint, you take the xpub at m/ at take the 4 bytes of hash160, which I believe you already did, but for `m/0'` instead.

Reply:
so rename to `get_accounts_fingerprint`?